hazelcast/src/main/resources/hazelcast-default.yaml

```diff
@@ -108,6 +108,12 @@ hazelcast:
       # Queue capacity. 0 means Integer.MAX_VALUE.
       queue-capacity: 0
 
+  scheduled-executor-service:
+    default:
+      capacity: 100
+      durability: 1
+      pool-size: 16
+
```

Comment:
Missing value for the `statistics` for this and `executor-service`. Also, I couldn't find an entry for the `durable-executor-service` in this file. Maybe we could add those

Reply:
done.

Reply:
`statistics-enabled: true` is still missing for the `executor-service` element

Reply:
should be fixed now.